#!/usr/bin/env python
PACKAGE = "tuw_iwos_odometer"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("acceleration_integration_iterations", int_t, 0, "number of odometer calculation iterations", 100, 0, 1000000)
gen.add("velocity_integration_iterations", int_t, 0, "number of odometer calculation iterations", 100, 0, 1000000)

exit(gen.generate(PACKAGE, "tuw_iwos_odometer", "ImuOdometer"))
